api/src/main/kotlin/edu/wgu/osmt/security/SecurityConfig.kt

```diff
@@ -26,6 +26,7 @@ import edu.wgu.osmt.RoutePaths.TASK_DETAIL_SKILLS
 import edu.wgu.osmt.RoutePaths.TASK_DETAIL_TEXT
 import edu.wgu.osmt.api.model.ApiError
 import edu.wgu.osmt.config.AppConfig
+import edu.wgu.osmt.db.WORKSPACE
```

Comment:
looks like this isn't needed

Reply:
oops it was supposed to be the routepath